examples/source/1.components/amp-accordion.html

```diff
@@ -3,6 +3,8 @@
 standaloneSnippets: true
 preview: true
 author: sebastianbenz
+transforms:
+  - email
```

Comment:
Can't we determine this automatically by transforming all samples and then running the validator?